docs/corsair-commander-core-guide.md

```diff
@@ -1,8 +1,10 @@
 # Corsair Commander Core, Core XT and ST
 _Driver API and source code available in [`liquidctl.driver.commander_core`](../liquidctl/driver/commander_core.py)._
 
+_New in 1.9.0: fixed duty cycles are now supported._<br>
 _Changed in 1.11.0: the Corsair Commander Core XT is now supported._<br>
 _Changed in 1.12.0: the Corsair Commander ST is now supported._<br>
+_New in 1.13.1: speed curve profiles are now supported._<br>
```

Comment:
Nit: I've generally opted to keep "New in" at the top of the sections describing the new feature. And as such they apply to the entire section, and need no further description.

In this case, it would be:

```
_New in 1.9.0._<br>
```

for the fixed speeds and

```
_New in git._<br>
```

for the newly suported speed curves.